swatch-api/deploy/clowdapp.yaml

```diff
@@ -96,56 +105,17 @@ objects:
           podSpec:
             image: ${IMAGE}:${IMAGE_TAG}
             initContainers:
-              - image: ${IMAGE}:${IMAGE_TAG}
-                env:
-                  - name: JAVA_DEBUG
-                    value: ${JAVA_DEBUG}
-                  - name: SPRING_PROFILES_ACTIVE
-                    value: liquibase-only
-                  - name: JAVA_MAX_MEM_RATIO
-                    value: '85'
-                  - name: GC_MAX_METASPACE_SIZE
-                    value: '256'
-                  - name: DATABASE_HOST
-                    valueFrom:
-                      secretKeyRef:
-                        name: swatch-tally-db
-                        key: db.host
-                  - name: DATABASE_PORT
-                    valueFrom:
-                      secretKeyRef:
-                        name: swatch-tally-db
-                        key: db.port
-                  - name: DATABASE_USERNAME
-                    valueFrom:
-                      secretKeyRef:
-                        name: swatch-tally-db
-                        key: db.user
-                  - name: DATABASE_PASSWORD
-                    valueFrom:
-                      secretKeyRef:
-                        name: swatch-tally-db
-                        key: db.password
-                  - name: DATABASE_DATABASE
-                    valueFrom:
-                      secretKeyRef:
-                        name: swatch-tally-db
-                        key: db.name
-                  - name: OTEL_EXPORTER_OTLP_ENDPOINT
-                    value: ${OTEL_EXPORTER_OTLP_ENDPOINT}
-                  - name: OTEL_EXPORTER_OTLP_PROTOCOL
-                    value: ${OTEL_EXPORTER_OTLP_PROTOCOL}
-                  - name: OTEL_SERVICE_NAME
-                    value: ${OTEL_SERVICE_NAME}
-                  - name: OTEL_JAVAAGENT_ENABLED
-                    value: ${OTEL_JAVAAGENT_ENABLED}
+              - image: ghcr.io/groundnuty/k8s-wait-for:v2.0
```

Comment:
We should not use images outside of our organization https://quay.io/organization/cloudservices. 
Also, taking into account that we have `dependencies: swatch-database`, should not this be enough? The deployment of this service should wait for the swatch-database deployment and hence the migrations already.

Reply:
`dependencies: swatch-database` doesn't work the way we need it to.  The other pods will wait for the ClowdApp to be successfully reconciled, but won't wait for the actual job to be complete.  You can remove the `initContainer` directives and do a deployment and see.

As for the external container.  Yes, either we will need to build a new container based on the wait-for script or else just mirror that container internally.